autograd_gamma/__init__.py

```diff
@@ -79,5 +91,11 @@ def _central_difference(ans, a, x):
 )
 
 
-reg_lower_inc_gamma = gammainc
-reg_upper_inc_gamma = gammaincc
+defvjp(
```

Comment:
this is the core addition. It defines how the derivative should be computed for each argument (argnum=0, 1 or 2). For the first two arguments, there isn't a analytical solution, so we use a finite difference method. For the last argument (w.r.t. x), we do have an analytical solution (this was copied from autograd)